docker/gremlin-server/gremlin-server-integration.yaml

```diff
@@ -18,8 +18,10 @@
 host: 0.0.0.0
 port: 45940
 evaluationTimeout: 30000
+channelizer: org.apache.tinkerpop.gremlin.server.channel.WsAndHttpChannelizer
```

Comment:
what is default channelizer, UnifiedChannelizer or WsAndHttpChannelizer?

Reply:
It is now WsAndHttpChannelizer, due to https://issues.apache.org/jira/browse/TINKERPOP-2765.

Reply:
Oh sorry, if you mean the default of Gremlin Server when you don't specify, I believe it's WebSocketChannelizer

Reply:
I asking because there is a bit different setup between
docker/gremlin-server/gremlin-server-integration-secure.yaml
docker/gremlin-server/gremlin-server-integration.yaml

but this should not affect our tests

Reply:
Oh no, it won't, the `WsAndHttpChannelizer` is for the `wait-for-server.sh` script, which needed an HTTP endpoint to ping. That only affects the no auth server.